src/main/java/frc/robot/ExtraMath.java

```diff
@@ -3,35 +3,36 @@
 public final class ExtraMath {
     private ExtraMath() {}
 
-    // https://www.desmos.com/calculator/kfyws5ap6u
+    // Forces a value into a set range
     public static double clamp(double val, double start, double end) {
         double min = Math.min(start, end);
         double max = Math.max(start, end);
-        if(val > max){
+        if (val > max) {
             val = max;
-        } else if(val < min){
+        } else if (val < min) {
             val = min;
         }
         return val;
     }
 
+    // Same as above but overloaded for ints
     public static int clamp(int val, int start, int end) {
         int min = Math.min(start, end);
         int max = Math.max(start, end);
-        if(val > max){
+        if (val > max) {
             val = max;
-        } else if(val < min){
+        } else if (val < min) {
             val = min;
         }
         return val;
     }
 
-    // https://stackoverflow.com/questions/5731863/mapping-a-numeric-range-onto-another
-    // https://www.desmos.com/calculator/0zisstnbzz
+    // Maps a value from one range to another
```

Comment:
I like your comment here, but I think the links to "where I got this from" and showing roughly what it does with desmos are a good idea to keep.